fdbserver/DataDistribution.actor.cpp

```diff
@@ -3722,9 +3987,18 @@ ACTOR Future<Void> dataDistributionTeamCollection(
 			self->redundantServerTeamRemover = serverTeamRemover(self);
 			self->addActor.send(self->redundantServerTeamRemover);
 		}
+
+		if (self->wrongStoreTypeRemover.isReady()) {
```

Comment:
just add to the addActor, do not keep a separate future, including for the above teamRemovers